src/module.ts

```diff
@@ -0,0 +1,41 @@
+import { defineNuxtModule } from '@nuxt/kit'
+import { join } from 'pathe'
+import { createFeedTemplate, resolveSources, writeFeedFile } from './feed'
+import type { FeedSource, FeedSourcesFactory } from './feed'
+
+export interface ModuleOptions {
+  sources: FeedSource[] | FeedSourcesFactory
+}
+
+export default defineNuxtModule<ModuleOptions>({
+  meta: {
+    name: '@nuxtjs/feed',
+    configKey: 'feed',
+    compatibility: {
+      nuxt: '^3.0.0',
+      bridge: false
+    }
+  },
+  defaults: {
+    sources: []
+  },
+  async setup (options, nuxt) {
+    const sources = await resolveSources(options.sources)
+
+    nuxt.hook('nitro:init', (nitro) => {
+      nitro.hooks.hook('close', async () => {
+        const routes = (nitro._prerenderedRoutes || []).filter(prerenderedRoute => prerenderedRoute.fileName?.endsWith('.html'))
```

Comment:
👀 `routes` is an empty array so that no `item` is added to feeds. Need some configuration in somewhere else for it to work?